### ISSUE TITLE
dataset link is broken

### ISSUE THREAD

**Issue by davyrouillard** [lbrejon/Localization-of-Asian-hornets-with-CNN-using-CAM#1]
Hello lbrejon, 
Thanks you for your project. I would like to try it but the hornet dataset link is broken. Can you put the dataset somewhere else?
Thanks you.
Davy

**Comment by chesnay** [comment]
Data not available anymore on google drive.

**Comment by davyrouillard** [comment]
Yes, I have seen :-). My question was: do you have a copy that you case share to use your software or perhaps is it become propriatory ?
Thank you,
Davy
